### ISSUE TITLE
More features in sidebar

### ISSUE THREAD

**Issue by rugk** [Lusito/web-ext-translator#11]
To solve our "conflict" in https://github.com/Lusito/web-ext-translator/issues/7 I have anotheri dea: Sure, put any preview feature you wish there, but more!
There is usually much more space when you have short strings.

E.g. you may display the description of the currently selected item there. I know, it is shown on hover, but 1) This does not work on touch devices and 2) This is not a thing the user usually finds out. One may need to know this.

Also you may integrate any other advanced features there later, you may come up with.

You could e.g. also limit the chars shown in the list and only show the full item on selection/in the sidebar, so one can better scroll through large values.

**Comment by Lusito** [comment]
Yeah, I wanted to show a list of available placeholders somewhere, so this is probably the place to put them.
